runtime/errors.rs

```diff
@@ -101,27 +100,6 @@ fn get_regex_error_class(error: &regex::Error) -> &'static str {
   }
 }
 
-fn get_request_error_class(error: &reqwest::Error) -> &'static str {
```

Comment:
Could we replace this with the `hyper::Error` handling?

Reply:
I've added a few for hyper, hyper-util, and hyper-v014. But just that they are `"Http"` errors...

Reply:
That should be fine then 👍